docs/java-rest/high-level/security/create-token.asciidoc

```diff
@@ -49,6 +49,7 @@ The returned `CreateTokenResponse` contains the following properties:
 `scope`:: The scope of the token. May be `null`.
 `refreshToken`:: A secondary "refresh" token that may be used to extend
  the life of an access token. May be `null`.
+`authentication`:: This is the authentication object for the newly created token.
```

Comment:
I think it is useful to cross-link the existing page of authentication response, e.g.:
 `<<{upid}-authenticate-response, authenticate response>>`. 
This applies to a few other similar places as well.

Reply:
Done